css/main.css

```diff
@@ -0,0 +1,303 @@
+    padding-left: 0px;
+    margin-top: 0px;
+    z-index: 1000000;
+    font-weight: bold;
+    font-size: 0.8em;
+    width: 100%;
+    background: #131313;
+    position: fixed;
+    text-align: center;
+}
+
+.menu ul {
+    margin: 0;
+    padding: 0!important;
+    list-style-type: none;
+    list-style-image: none;
+}
+
+.menu li {
+    display: block;
+    padding: 15px 0 15px 0;
+}
+
+.menu ul li a {
+    text-decoration: none;
+    text-align: center;
+    margin: 0px;
+    color: #fff;
+}
+
+.menu ul li a:hover {
+    color: #fff;
+    text-decoration: none;
+}
+
+.menu a {
+    text-decoration: none;
+    color: white;
+}
+
+.menu a:hover {
+    text-decoration: none;
+    color: white;
+}
+
+.cross {
+    display: none;
+}
+
+
+/*************** body elemnt ***************/
+
+.container {
+    margin-top: 160px;
+}
+
+.menu {
+    display: none;
+}
+
+.glyphicon-home {
+    color: white;
+    font-size: 1.5em;
+    margin-top: 5px;
+    margin: 0 auto;
+}
+
+
+/*	allscritpet*/
+
+a {
+    text-decoration: none;
+}
+
+h1 {
+    margin: 0px;
+    font-size: 3em;
+    color: #EB0951;
+}
+
+p,
+li {
+    color: #97866F;
+    padding: 0px 15px 0px 15px;
+}
+
+.pink {
+    color: #EB0951;
+}
+
+
+/* form script*/
+
+#searchForm {
+    display: none;
+}
+
+
+/*header script*/
+
+.img-center {
+    line-height: 10px;
+}
+
+.img-logo {}
+
+.img-logo h2 {
+    display: none;
+}
+
+.img-logo h1 {
+    margin-left: 20px;
+    font-stretch: extra-condensed;
+}
+
+.img-center {
+    background-color: #e4dace;
+    border: 1px solid #C3BDB5;
+    display: block;
+    height: 70px;
+    margin: 10px auto auto;
+    overflow: hidden;
+    text-align: left;
+}
+
+.img-center h2 {
+    color: #FDFAF8;
+}
+
+.img-center h1 {
+    color: #EFECE3;
+}
+
+.img-center h1,
+.img-center h2 {
+    line-height: 1em;
+    margin-top: 0;
+    text-transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    -ms-transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    /* IE 9 */
+    -webkit-transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    /* Chrome, Safari, Opera */
+    transform: rotate(-8deg) translateY(-10px) translateX(0px);
+    white-space: nowrap;
+}
+
+
+/*body element*/
+
+.column-1 {
+    margin-top: 80px;
+}
+
+.column-1 img {
+    width: 100%;
+    margin: auto;
+    display: block;
+}
+
+.column-2 h1,
+.column-2 h2 {
+    text-align: center;
+}
+
+.column-2 p {
+    text-align: justify;
+}
+
+.right-block {
+    background-color: #FFFFFF;
+    border-radius: 10px;
+    margin-top: 40px;
+}
+
+.right-block h3 {
+    color: #EB0951;
+    padding: 15px 10px 15px 23px;
+    text-align: center;
+    border-bottom: dotted #EAE2D7;
+}
+
+.right-block a {
+    color: black;
+    height: 40px;
+    line-height: 40px;
+    text-decoration: none;
+    margin-right: 10px;
+}
+
+#follow-us a {
+    color: black;
+    line-height: 35px;
+    display: block;
+    float: none;
+    margin: auto;
+    width: 130px;
+}
+
+.right-block img {
```

Comment:
What will happen if you'll add one more block with an image which should not have any margin?